Docs/source/Software/Drivers.rst

```diff
@@ -17,3 +17,29 @@ Additional Considerations
     The EEPROM is a non-volatile memory chip, so any previous data on the EEPROM will remain unless ``EEPROM_Reset()`` is called. The EEPROM is rated for
     4 million write cycles, so we should try use less write cycles than this over the lifetime of the BPS. It takes around 5 milliseconds to write to the 
     EEPROM, so multiple writes should not be attempted within the same 5 ms time period, or some of the writes may fail.
+
+CAN Driver
+===========
+
+Purpose
+    This driver is meant to send and recieve CAN messages. This allows us to talk with other systems
+    in the car.
+
+Usage
+    The header file contains data types that are used to send CAN messages. Each ``CANMSG_t`` should 
+    be sent with a ``CANID_t`` and ``CANPayload_t`` initialized in a struct. ``CANPayload_t`` should
+    contain the data (``CANData_t``) and id of the value if it is in an array. If it isn't in an array
+    ``idx`` should be 0. ``''CANData_t'' should only have the value of the data type being sent 
+    filled in into the struct. For example, if the data is a float, only fill in ``CANData.f`` and 
+    nothing else. There are two types of sending and recieving functions one can use. One is 
```

Comment:
I think this would be easier to read if there was a newline right before "There are two types..."